src/Microsoft.IdentityModel.JsonWebTokens/JsonWebToken.cs

```diff
@@ -154,7 +154,7 @@ public IEnumerable<string> Audiences
         /// <summary>
         /// Gets a <see cref="IEnumerable{Claim}"/><see cref="Claim"/> for each JSON { name, value }.
         /// </summary>
-        public virtual IEnumerable<Claim> Claims
```

Comment:
>virtual [](start = 15, length = 7)

This is technically a breaking change in the public API, while small someone might have made an override of this method. This will cause some version update requirements etc... alternatively can the virtual stay?